lib/Partitioner/PartitionerUtils.cpp

```diff
@@ -272,4 +272,16 @@ GraphMemInfo getGraphMemInfo(const NodesSetTy &nodes) {
   return ret;
 }
 
+std::set<Kinded::Kind> generateNonSupportedNodes(llvm::StringRef names) {
+  std::set<Kinded::Kind> ret;
+  while (names.find(',') != std::string::npos) {
+    std::pair<llvm::StringRef, llvm::StringRef> newSplit = names.split(',');
```

Comment:
Probably not very important, but could it be that split allocates new strings? Please have a look at the similar function for splitting comma-separated input here:

https://llvm.org/doxygen/CommandLine_8cpp_source.html#l00607

Reply:
/me wishes for `folly::split` (but of course we don't want to pull in folly, alas)